src/typeGuards/index.ts

```diff
@@ -7,4 +7,3 @@
  */
```

Comment:
If you don't need anymore i would just delete this folder

Reply:
Sure, deleting it.